### ISSUE TITLE
Brackets Crash

### ISSUE THREAD

**Issue by DigitalFeonix** [mikaeljorhult/brackets-todo#143]
When I use the Windows command prompt to run a PHP file and pipe it to a log file within the project file scope, it will crash Brackets (the window goes gray and "Brackets has stopped working" dialog appears).

Confirmed with testing that it is the Todo extension alone - no other user extensions loaded - that is doing this. The log file doesn't have to be open either.

### Steps to Reproduce

1. Use a script that outputs something and pipe to a file

```
<?php

echo "We're going to break Brackets now\n";
```

`C:\dev\CLI scripts>php -f break-brackets.php > break.log`

2. Watch Brackets crash

### Versions

Todo 0.9.4
Brackets 1.10 32-bit on Windows 7

**Comment by DigitalFeonix** [comment]
I updated Todo and re-ran my tests with the same results.

I can give some more info though. When piping to the log file, if it does not exist yet or you are appending then it does not crash. Only when outputing to a file that already exists then it will crash.

will not crash Brackets if file does not exist
`C:\dev\CLI scripts>php -f break-brackets.php > break.log`

will crash Brackets if file already exists
`C:\dev\CLI scripts>php -f break-brackets.php > break.log`

will not crash Brackets
`C:\dev\CLI scripts>php -f break-brackets.php >> break.log`

### Versions

Todo 0.9.5
Brackets 1.10 32-bit on Windows 7

**Comment by mikaeljorhult** [comment]
Yes, I have noticed the same thing but haven't found a fix for it yet. Thank you for confirming the results!

**Comment by brigevans** [comment]
Also tried to exclude .log file, the file name itself, and also the logs folder, and none of them corrected the issue with the crashing on log files. 
![image](https://user-images.githubusercontent.com/2281944/30561477-67557c96-9c78-11e7-984e-3081adefc9e3.png)
